src/bluehawk/actions/copy.ts

```diff
@@ -33,6 +34,24 @@ export const copy = async (
     });
     return;
   }
+
+  let renameObj = {};
+  if (typeof rename === "string") {
+    try {
+      renameObj = JSON.parse(rename);
+    } catch {
+      throw "Unable to parse 'rename' argument. Ensure your 'rename' argument is valid JSON.";
+    }
+  }
+
+  // function to check if object has a new name specified
+  const getRename = (fileName: string) => {
+    if (rename && renameObj && (renameObj as any)[fileName] !== undefined) {
```

Comment:
Watch out for casting to `any`. This could be avoided if the parsing of the input were done in the interface layer (CLI) rather than action.

Reply:
Updated type of rename to a `Record<String, String>` to avoid `any` type cast. Good catch!